package.json

```diff
@@ -53,5 +53,18 @@
   },
   "lint-staged": {
     "*.{js,json,md}": ["prettier --write", "git add"]
+  },
+  "jest": {
+    "collectCoverage": true,
+    "coverageDirectory": "coverage",
+    "collectCoverageFrom": ["src/**/*.js"],
+    "coverageThreshold": {
+      "global": {
+        "lines": 82,
+        "statements": 80,
+        "functions": 90,
+        "branches": 50
+      }
+    }
```

Comment:
Are these not the defaults? You could just pass `--coverage` to the command

Reply:
Good call! 👍  I removed the default configuration, leaving the threshold configuration to break the build in case it decreases with other contributions. They're still available to be increased/decreased in case the changes in the future